.node-version

```diff
@@ -0,0 +1 @@
+14.16.1
```

Comment:
The current LTS; Some of the deps are fairly old and might need another pass once v16 is active is LTS. You can probably wait until 14 is EOL though, which is a couple of years.